plugins/network-elements/tungsten/src/main/java/com/cloud/network/guru/TungstenGuestNetworkGuru.java

```diff
@@ -14,7 +14,7 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-package org.apache.cloudstack.network.tungsten.service;
+package com.cloud.network.guru;
```

Comment:
this move does not seem right. are you sure you want this? nothing new should go under `com.cloud`

Reply:
Could you please explain me why we can not have new in com.cloud ?
I saw some plugin like BigSwitchBcf, BrocadeVcs, cisco, ElasticLb, ... in com.cloud. And they have unit test for NetworkGuru.
While some plugin like Opendaylight, SspGuest not in com.cloud. And they have no unit test for NetworkGuru.
So does it have any difference between them ?

Reply:
com.cloud is an old company that donated the larger part of the system (via via) to apache. new code should be under org.apache.cloudstack. Unless you are working for com.cloud please move it.

Reply:
Thanks for your explanation. I will roll it back in the next commit.

Reply:
thanks @huyle-uni